lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -659,21 +664,21 @@ LogicalResult CppEmitter::emitAttribute(Attribute attr) {
   }
   if (auto sAttr = attr.dyn_cast<SymbolRefAttr>()) {
     if (sAttr.getNestedReferences().size() > 1) {
-      return failure();
+      return op.emitError(" attribute has more than 1 nested references");
     }
     os << sAttr.getRootReference();
     return success();
   }
   if (auto type = attr.dyn_cast<TypeAttr>()) {
-    return emitType(type.getValue());
+    return emitType(op, type.getValue());
   }
-  return failure();
+  return op.emitError("cannot emit attribute of type ") << attr.getType();
```

Comment:
Can we add a test for this or is this one of the untestable errors?

Reply:
This code shouldn't be reached, unless we missed a possible attribute type. So currently there is no known way to produce and test this error.

Reply:
Okay :)